rbac/management/workspace/serializer.py

```diff
@@ -28,12 +28,21 @@ class WorkspaceSerializer(serializers.ModelSerializer):
     uuid = serializers.UUIDField(read_only=True, required=False)
     description = serializers.CharField(allow_null=True, required=False, max_length=255)
     parent_id = serializers.UUIDField(allow_null=True, required=False)
+    created = serializers.DateTimeField(read_only=True)
+    modified = serializers.DateTimeField(read_only=True)
 
     class Meta:
         """Metadata for the serializer."""
 
         model = Workspace
-        fields = ("name", "uuid", "parent_id", "description")
+        fields = (
```

Comment:
'__all__' will also work in this case 😁

Reply:
Right, sounds good, just leave it as it is 👍

Reply:
Thanks for the feedback!